platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/item/message/WorkflowItemMessageRowsLoader.java

```diff
@@ -128,7 +128,7 @@ private WorkflowItemMessageRow createAutoTransitionSourceRow(AGWorkflowAutoTrans
 
 		return createTransitionRow(//
 			transitionExecution.getWorkflowTransition().getSourceNode(),
-			transitionExecution.getTransaction());
+			transitionExecution.getTransaction()).setIsAutoTransitionMessage(true);
```

Comment:
- This won't work as expected, it will make the last auto transition seem to be done by the user and the first transition to be done by the system user, which is both wrong. You can also see this in the second screenshot.
- We need to talk.